cidc_api/gcloud_client.py

```diff
@@ -213,15 +213,15 @@ def publish_artifact_upload(file_id: int):
         report.result()
 
 
-def send_email(to_emails: List[str], subject: str, html_content: str):
+def send_email(to_emails: List[str], subject: str, html_content: str, **kw):
```

Comment:
Maybe add a note in the docstring that `**kw` are any sendgrid client-supported email config keyword arguments?